j2cl-tasks/src/main/java/com/vertispan/j2cl/tools/Javac.java

```diff
@@ -95,4 +104,8 @@ public boolean compile(List<FileInfo> modifiedJavaFiles) {
             });
         }
     }
+
+    public enum Mode {
```

Comment:
The more branches that go into this class the less I like having it at all, since this one class specializes in several ways for its two specific use cases - but I think for the short term this is fine.

Reply:
:-)